packages/component/src/Activity2/Timestamp.js

```diff
@@ -0,0 +1,83 @@
+import { connect } from 'react-redux';
+import React from 'react';
+
+import String, { getString } from '../Localization/String';
+import Context from '../Context';
+import TimeAgo from '../Utils/TimeAgo';
+
+// TODO: We could refactor this into a general component
+function sendFailed(language, replace) {
+  const text = getString('Send failed, {retry}', language);
+  const retry = getString('retry', language);
+  const match = /\{retry\}/.exec(text);
+
+  if (match) {
```

Comment:
is this string.format?

Reply:
Added comments to `en-US.js` to indicate `{retry}` should not be localized.

I am lazy here, should write a more generic `getString` that do recursive translations based on curly-braces. But I will leave it as-is for now as this is the only string that requires placeholder.